code/weapon/beam.cpp

```diff
@@ -2437,7 +2437,7 @@ void beam_jitter_aim(beam *b, float aim)
 	vm_vector_2_matrix(&m, &forward, NULL, NULL);
 
 	// get a vector on the circle - this should appear to be pretty random
```

Comment:
Edit the comment to clarify it's random with a bias.  I know the `true` parameter should suffice but people might read just the comment and not look at that.